### ISSUE TITLE
Add share/action button to video page

### ISSUE THREAD

**Issue by raphaelschaad** [raphaelschaad/Plus-Web#36]
Has to easily support the notion of "Send to myself as email"

**Comment by xaverfleer** [comment]
looks easy to me: ![share](https://user-images.githubusercontent.com/7235734/62889108-f6f02280-bd40-11e9-9ddb-3cae852ea402.png)

Question: How can I access our Wistia account? I does not seem to be a [feature to pay for](https://xaver-fleer.wistia.com/account/subscriptions)..

**Comment by raphaelschaad** [comment]
Good point, we could use [the built-in Wistia feature](https://wistia.com/support/embed-and-share/sharing-videos). There are a few product design questions to quickly think through:
- Is a built-in share feature discoverable enough for the situation where someone receives the video by text to their phone and thinks "I want to easily forward it to my email instead to watch it on a big screen" or does it need a more obvious, Plus-specific Prompt (a la "Senden Sie Ihr Video an Ihre E-Mail")?
- Do we need a mechanism for throttling sharing?
- Should we track sharing?
- If the route to the patient video gets protected (i.e. behind patient "login"), does the built-in share feature still work as expected?
- Anything I haven't thought of yet.
